lib/timers.js

```diff
@@ -94,8 +94,8 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 //
 // - key = time in milliseconds
 // - value = linked list
-const refedLists = {};
-const unrefedLists = {};
+const refedLists = Object.create(null);
```

Comment:
@fhinkel Just curious. Do you mean using a Map instead of Object.create? https://github.com/nodejs/node/pull/7581#issuecomment-236117659